build.gradle

```diff
@@ -80,6 +80,7 @@ dependencies {
     }
     testCompile group: 'org.mockito', name: 'mockito-all', version: mockitoVersion
     testCompile group: 'org.apache.hadoop', name: 'hadoop-common', version: hadoopVersion
+    testCompile group: 'com.github.stefanbirkner', name: 'system-rules', version: junitSystemRulesVersion
```

Comment:
The new component uses [Common Public License](https://github.com/stefanbirkner/system-rules/blob/master/LICENSE). Open Source Initiative lists the CPL as deprecated and superseded by EPL, as per [Wikipedia](https://en.wikipedia.org/wiki/Common_Public_License). It might be a good idea to double-check if it is OK to use a component with this license (I believe it should be OK).

Reply:
I have removed the package now since we are also passing the same configurations as system properties.